content/500-platform/10-about.mdx

```diff
@@ -5,13 +5,132 @@ metaDescription: 'Learn about the Platform Console to integrate the Prisma Data
+- [**Environments**](#environment): An environment belongs to a project. It typically maps to a _development stage_, like `dev`, `staging` or `prod`. **API keys are provisioned on the environment level**, and products are configured per environment as well (e.g. the database connection string used for Accelerate).
+
+Here is a visual illustration of how these concepts relate to each other:
+
+![](/img/platform/pdp-concepts.png)
+
+### User account
+
+A user account is the prerequisite for any interactions with PDP products. You can use it to manage your workspaces (and their projects). A user account can be invited to collaborate on workspaces created by other users as well.
+
+If you need to delete your user account, go [here](/platform/support#deleting-your-pdp-account).
+
+### Workspace
+
+You can create several workspaces. A workspace is an isolated space to host projects. A workspace can have multiple user accounts associated with it so that multiple users can collaborate on the the projects in the workspace.
+
+In each workspace, you can:
+
+- view and manage all projects (and their environments) in that workspace.
+- manage billing, i.e. select a [pricing plan](https://prisma.io/pricing?utm_source=prisma-docs&utm_medium=docs-site), configure payment methods, or view the invoice history.
+- view the usage of your enabled PDP products across all projects in that workspace.
+- manage team members and change their roles / permissions in the workspace.
+
+### Project
+
+In each workspace, you can create several projects. A project typically represents an application (a product or service). You typically have one [Prisma schema](/orm/prisma-schema/) per project.
+
+In each project, you can:
+
+- view and manage all environments in that project.
+
+The number of project you can create in a workspace depends on the [pricing plan](https://prisma.io/pricing?utm_source=prisma-docs&utm_medium=docs-site) configured in that workspace:
+
+Here's an overview of the number of projects per workspace on each plan:
+
+|              | **Starter** | **Pro** | **Business** | **Enterprise** |
+| ------------ | ----------- | ------- | ------------ | -------------- |
+| **Projects** | 5           | 10      | 15           | Custom         |
+
+### Environment
+
+An environment is an isolated space to provision PDP products for a given project. Environments typically map to development stages, like `dev`, `staging` or `prod`.
+
+In each environment, you can:
+
+- enable, disable and configure PDP products (Pulse, Accelerate, ...).
+- for **Accelerate**:
+  - set your database connection string.
+  - configure the _region_ where Accelerate's connection pool is running.
+  - generate API keys.
+  - change the connection pool size.
+  - enable static IP.
+- for **Pulse**:
+  - set your database connection string (of a [Pulse-compatible database](/pulse/database-setup)).
```

Comment:
```suggestion
  - set your database connection string.
```

Reading this again, I think this is implied. You can also only use a compatible connection string for Accelerate, but we don't call it out.